### PR TITLE
Update maint/maint-68 to 68.2.0.2

### PR DESCRIPTION
**Pull request by jefgen**
This updates the `maint/maint-68` branch to 68.2.0.2
(This will be done as a merge commit to keep the history).

## PR Checklist
* [x] I have verified that my change is specific to this fork and cannot be made upstream.
* [x] I am making a maintenance related change.
* [ ] I am making a change that is related to usage internal to Microsoft.
* [ ] I am making a change that is related to the Windows OS build of ICU.
* [x] CLA signed. If not, please see [here](https://cla.opensource.microsoft.com/microsoft/icu) to sign the CLA.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
 # Changelog
 
-## ICU 68.1.0.1
+## ICU 68.2.0.2
+#### Changes cherry-picked from upstream tickets/PRs:
+
+ICU-21465 Windows Time Zone offset is wrong when Automatic DST setting is OFF
+- https://unicode-org.atlassian.net/browse/ICU-21465
+- https://github.com/unicode-org/icu/pull/1465
```
Comment (daniel-ju) [inline changelog.md:8]: Should this be `https://github.com/unicode-org/icu/pull/1539` instead?
(Sorry I didn't notice this in the earlier PR into master.)

Comment (jefgen) [reply changelog.md:8]: Ah,.. good catch!

Yes this should really be `https://github.com/unicode-org/icu/pull/1539` instead.
It seems that I also made the same mistake in the PR here as well. https://github.com/microsoft/icu/pull/65 :/

Sigh, I can't force-push to fix the existing commit though...
But I'll fix up the PR description and the changelog.md file.

Comment (jefgen) [reply changelog.md]: Fixed with the latest commit. :)

Comment (daniel-ju) [reply changelog.md]: Thanks! :)

**Review by daniel-ju — APPROVED** [review]

**Pull request merged**